apps/fishing-map/features/vessel-groups/vessel-groups.slice.ts

```diff
@@ -128,7 +128,7 @@ export const searchVesselGroupsVesselsThunk = createAsyncThunk(
             message: 'Missing search url',
           })
         }
-        const searchResults = await GFWAPI.fetch<APIPagination<VesselSearch>>(url, { signal })
+        const searchResults = await GFWAPI.fetch<APIPagination<VesselSearch>>(url, { signal, version: '' })
```

Comment:
please see comment below, I think we could avoid the `version:''` where the urls are generated by `resolveEndpoint`